api/examples/kafka-ephemeral-ingress.yaml

```diff
@@ -19,22 +19,241 @@ spec:
           brokers:
           - broker: 0
             host: broker-0.${cluster_name}.${kube_ip}.nip.io
-#          - broker: 1
-#            host: broker-1.${cluster_name}.${kube_ip}.nip.io
+          - broker: 1
+            host: broker-1.${cluster_name}.${kube_ip}.nip.io
+          - broker: 2
+            host: broker-2.${cluster_name}.${kube_ip}.nip.io
+      - name: oauth
+        port: 9093
+        type: ingress
+        tls: true
+        configuration:
+          bootstrap:
+            host: bootstrap-oauth.${cluster_name}.${kube_ip}.nip.io
+            annotations:
+              "eyefloaters.github.com/console-listener": "true"
```

Comment:
Anlong similar lines to the comment about doc/`README.md`, I wonder if a few comments in here would be useful to those wanting to understand this example?

Reply:
> Anlong similar lines to the comment about doc/`README.md`, I wonder if a few comments in here would be useful to those wanting to understand this example?

We can also take care of adding comments in the examples resources in #63